_posts/2020-10-04-newpipe-0.20.0.md

```diff
@@ -1,154 +1,176 @@
 
 ### Android 9
 
-First of all, we say "sorry" to our users on Android 9 and below. The progress bar is gone. As we've said above, it will only be shown on Android 10 and above. We're a small team and can't maintain code for the old and new types of notifications simultaneously, so the old one has to go. And not just that; the old notification isn't even supported Android 11 onwards. Bye-bye RemoteView notifications. We shall miss you! :'(
-
-But you get a lot in exchange for that sacrifice!
+Android 9 is not Android 10, and there is now only a progress bar on 10. It makes sense to keep things managable by our small team. The old notification isn't even supported on Android 11 onwards. RemoteView notifications are out, MediaStyle is in.
+Your sacrifice is not in vain:
 
 ### New features
 
-You get five CUSTOMISABLE - you read it right! - **CUSTOMISABLE** action buttons to control playback. There is a new <code><img src="/img/icons/play_arrow-24dp.svg" /> notification</code> category in the app settings, and you can choose which button should perform which player action. Play/pause, forward/rewind, previous/next, shuffle, repeat, close, or even nothing!
+Button up or button down, you get five **customisable** action buttons to control playback. Batteries not included. There is a new <code><img src="/img/icons/play_arrow-24dp.svg" /> notification</code> category in the app settings, from where you can pick actions and the buttons to carry out your bidding. Play/pause, forward/rewind, previous/next, shuffle, repeat, close, or even nothing! The decadency.
 
-Then, you can choose which three of these five buttons show in the compact notification (remember that little arrow you use to expand/contract the notification?).
+Three of these five buttons can be shown in the compact notification (remember that little arrow you use to expand/contract the notification?).
 
-For users on **Android 8+**: Colours! Warm colours. Cool colours. All the colours! As we've noted above, MediaStyle notifications adapt to the colours of the cover art, which looks just plain awesome.
+For the betrodden of **Android 8+**: Colours! Warm colours. Cool colours. All the colours! As noted above, MediaStyle notifications adapt to cover art colours. Big.
 
-For users on **Android 10+**: Seekbar! The old notification had a progress bar, but that was just a visual indicator. Now you can actually seek from the notification itself. Marvel at the massive boost in convenience, and have fun seeking!
+For the modern human using **Android 10+**: SeekBar! The old notification had a progress bar, but that was just a visual indicator. Now you can actually seek from the notification itself. Marvel at the massive boost in convenience, and you shall find what you seek.
 
-For users on **Android 11**: Congratulations! You get a notification now! NewPipe's media session will show up in the notification shade and in Quick Settings just as advertised.
```

Comment:
Triple "for users on". Do something with it. Double shouting at the reader. "media session"? unreferenced Quick Settings. Advertised where?

Reply:
> Triple "for users on". Do something with it.

What did TheAssassin say? Anaphora? Anaphora. 😎

> Double shouting at the reader. 

Exclamation marks aren't shouting. All caps is shouting. Refer to sales pitch joke above.

> "media session"?

Android 11 marketing terminology. We just use it as is.

> unreferenced Quick Settings. Advertised where?

By Google in all Android 11 marketing material.

Reply:
This is "mimicking" spam, which is to say it is spam. Android 11 marketing material doesn't change how language works for everyone else. It isn't a sales pitch joke if you can't pull it off, it is a joke of a sales pitch.